README.md

```diff
@@ -24,3 +24,20 @@ Imagine that we can fetch from an API the details of all movies that were ever s
 **License**: `[<start year of the license>: Number, <end year of the license>: Number]`
 
 Write a JS Module that returns in what year we had the maximum number of movies on that platfrom
+
+
+# Solution Complexity
+
+### Time Complexity
+
+`O(L*dmax)+dmax` : Where L is the length of the movies array and dmax is the maximum difference in the years                     of the movie licenses in the movies array
```

Comment:
L*dmax + dmax = L * (dmax + 1) = L * dmax

So, complexity is `O(L * dmax)`

Reply:
Why is the complexity still dependent on `dmax`

Did you not work on the solution that makes it free of what the data years are, and instead keeps it dependent only on the size of the data?